hubble-be/src/main/java/com/baidu/hugegraph/service/load/LoadTaskService.java

```diff
@@ -143,59 +135,60 @@ public int taskCountByJob(int jobId) {
     }
 
     @Transactional(isolation = Isolation.READ_COMMITTED)
-    public int save(LoadTask entity) {
-        return this.mapper.insert(entity);
+    public void save(LoadTask entity) {
+        if (this.mapper.insert(entity) != 1) {
+            throw new InternalException("entity.insert.failed", entity);
+        }
     }
 
     @Transactional(isolation = Isolation.READ_COMMITTED)
-    public int update(LoadTask entity) {
-        return this.mapper.updateById(entity);
+    public void update(LoadTask entity) {
+        if (this.mapper.updateById(entity) != 1) {
+            throw new InternalException("entity.update.failed", entity);
+        }
     }
 
     @Transactional(isolation = Isolation.READ_COMMITTED)
-    public int remove(int id) {
-        this.taskContainer.remove(id);
-        return this.mapper.deleteById(id);
+    public void remove(int id) {
+        this.runningTaskContainer.remove(id);
+        if (this.mapper.deleteById(id) != 1) {
+            throw new InternalException("entity.delete.failed", id);
+        }
     }
 
-    public List<LoadTask> batchTasks(int job_id) {
+    public List<LoadTask> batchTasks(int jobId) {
         QueryWrapper<LoadTask> query = Wrappers.query();
-        query.eq("job_id", job_id);
+        query.eq("job_id", jobId);
         return this.mapper.selectList(query);
     }
 
     public LoadTask start(GraphConnection connection, FileMapping fileMapping) {
-        connection = sslService.configSSL(this.config, connection);
+        connection = this.sslService.configSSL(this.config, connection);
         LoadTask task = this.buildLoadTask(connection, fileMapping);
-        if (this.save(task) != 1) {
-            throw new InternalException("entity.insert.failed", task);
-        }
-        this.taskExecutor.execute(task);
-
-        if (this.update(task) != 1) {
-            throw new InternalException("entity.update.failed", task);
-        }
+        this.save(task);
+        // executed in other threads
+        this.taskExecutor.execute(task, () -> this.update(task));
```

Comment:
should handle exception if the update is failed?

Reply:
no need, it doesn't affect other tasks to use the thread pool